CONTRIBUTING.md

```diff
@@ -9,7 +9,7 @@ community.
 ### Participation Conduct
 
 In order to ensure everyone has a fair, pleasant, and inclusive experience contributing to Neutrino, we ask that you
-abide by our [community participation guidelines](/contributing/code-of-conduct.md), based on the
+abide by our [community participation guidelines](/docs/contributing/code-of-conduct.md), based on the
```

Comment:
Links for gitbook have to be relative in order for them to work on the docs site as well as github. Change to: `./code-of-conduct.md`.